public/locales/en/translation.json

```diff
@@ -855,7 +855,7 @@
     "title": "Settings",
     "general": "General",
     "actionItemCategories": "Action Item Categories",
-    "updateOrganization": "Update Organization",
+    "Edit Organization": "Edit Organization",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.